mods/parsoid.js

```diff
@@ -498,16 +498,42 @@ PSP._getOriginalContent = function(restbase, req, revision, tid) {
 
 };
 
+PSP._getStashedContent = function(restbase, req, etag) {
```

Comment:
Eventually we should be able to merge this with _getOriginalContent, but for now this is the only one loading wikitext.